src/style.css

```diff
@@ -0,0 +1,102 @@
+@font-face {
+  font-family: "Avenir-Book";
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  font-weight: 900;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  font-family: "Avenir-Book", sans-serif;
+  font-weight: 300;
+}
+
+.bigFoarm {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.input-container {
+  position: relative;
+}
+
+img {
```

Comment:
use classnames instead of tag selector

Reply:
is not fixed.
there are still using tag selectors for adding styles (pay attention to input)